spec/ddtrace/configuration_spec.rb

```diff
@@ -210,12 +211,12 @@
         end
 
         context 'have one of a few replaced' do
-          let(:old_statsd) { instance_double(Datadog::Statsd) }
-          let(:new_statsd) { instance_double(Datadog::Statsd) }
+          let(:old_statsd) { instance_double(Datadog::Statsd, close: nil) }
+          let(:new_statsd) { instance_double(Datadog::Statsd, close: nil) }
 
           before do
             # Since its being reused, it should not be closed.
-            expect(old_statsd).to_not receive(:close)
+            allow(old_statsd).to receive(:close)
```

Comment:
I believe the `allow` can be removed, since adding `close: nil` to the double implicitly states that it is allowed.